src/main/java/org/usfirst/frc4904/robot/commands/GroundIntake.java

```diff
@@ -0,0 +1,17 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Manipulator;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import org.usfirst.frc4904.standard.commands.solenoid.SolenoidExtend;
+import org.usfirst.frc4904.standard.commands.solenoid.SolenoidRetract;
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class GroundIntake extends CommandGroup {
+	public GroundIntake() {
+		addSequential(new SolenoidExtend("ManipulatorGroundOut", RobotMap.Component.manipulator.ground));
```

Comment:
Since you use ManipulatorGroundOut several times, it makes sense to make a Command called ManipulatorGroundOut that extends SolenoidExtend that passes in these arguments. That way, to edit the command, you only need to edit it once. Do the same for all other SolenoidExtend/SolenoidRetract/SolenoidOff commands that are used multiple times